stdlib/public/core/FloatingPointTypes.swift.gyb

```diff
@@ -1797,6 +1797,15 @@ extension ${Self} {
     return (lhs + rhs, { v in (v, v) })
   }
 
+  @inlinable // FIXME(sil-serialize-all)
+  @_transparent
+  @differentiating(+)
```

Comment:
We should not use `@differentiating` here. Instead, add a `jvp:` to the `@differentiable` attribute declared on `func +`.

Reply:
Actually nvm since the existing code is already using it. I'm surprised this didn't cause the linking error I've seen earlier.